doc/modules/tree.rst

```diff
@@ -531,6 +535,25 @@ Log Loss or Entropy:
 
       \mathrm{LL}(D, T) = \sum_{m \in T} \frac{n_m}{n} H(Q_m)
 
+**HellingerDistance**
+
+Hellinger distance is implemented for the binary case only and a single label.
+
+.. math::
+
+    H(Q_m) = \sqrt{\left(\sqrt{\frac{N_{m,+}^{left}}{N_{m,+}}}-\sqrt{\frac{N_{m,-}^{left}}{N_{m,-}}}\right)^2+\left(\sqrt{\frac{N_{m,+}^{right}}{N_{m,+}}}-\sqrt{\frac{N_{m,-}^{right}}{N_{m,-}}}\right)^2 }
+
+where :math:`N_{m,+}` and :math:`N_{m,-}` are the number of positive and
+negative samples for the :math:`m` node respectively; :math:`N_{m,+}^{left}` and
+:math:`N_{m,+}^{right}` are the number of positive samples for each of the
+:math:`m` node splits. Similarly, :math:`N_{m,-}^{left}` and
+:math:`N_{m,-}^{right}` are the number of negative samples for each of the
+:math:`m` node splits.
```

Comment:
I think this would read better if it were formatted as a list.

Reply:
looked in this documentation file, looks like other section explain the formulas in same format
can you please share an example of a list format in the documentation that will help me understand what you had in mind?

Reply:
[This section of on constraints of monotonicity for HGBT](https://scikit-learn.org/stable/modules/ensemble.html#monotonic-constraints) feature a bullet-point list and mathematical notations:
https://github.com/scikit-learn/scikit-learn/blob/a0a6ea744f355ed078e491eadf49d1b9c1a17cda/doc/modules/ensemble.rst?plain=1#L1136-L1149

I am trying to find one used in docstrings using bullet-point list and inline mathematical notations but I have not been able to find one yet. Still, I think such formatting structure can easily be inferred from this example.

Let me know if this is not clear.

Reply:
thanks for the example
I followed it and formatted the formula description as a list